client/index.jsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import axios from 'axios';
+import ReviewsRender from './components/ReviewsRender';
+
+class App extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      reviews: [],
+      reviewCount: 3,
```

Comment:
@Matt-Wigg , what does reviewCount do? Changes the number of reviews rendered in the component? 

If that's the case, how many reviews do you fetch from the backend in a single request? And do you have a pagination mechanism to fetch more reviews once all reviews on the client side are rendered?

Reply:
Hey @miraclestyle - review count does exactly that! It's a way for me to specify the amount of reviews displayed at one time. I pull all reviews related to a particular product Id from my initial get request, then do the filtering client side.

Reply:
I understand. This will, perhaps, be a good opportunity to work on during the optimization phase.